src/main/resources/db/migration/courtcase/V109__make_fk_from_offence_to_court_case.sql

```diff
@@ -5,8 +5,14 @@ ALTER TABLE OFFENCE DROP COLUMN case_id;
 ALTER TABLE COURT_CASE DROP CONSTRAINT court_case_case_id_idempotent;
 
 -- Add columns to make the FK
-ALTER TABLE OFFENCE ADD COLUMN CASE_NO TEXT NOT NULL;
-ALTER TABLE OFFENCE ADD COLUMN COURT_CODE TEXT NOT NULL;
+ALTER TABLE OFFENCE ADD COLUMN CASE_NO TEXT;
+ALTER TABLE OFFENCE ADD COLUMN COURT_CODE TEXT;
+
+DELETE FROM OFFENCE WHERE ID > 1;
```

Comment:
On this line I think you could alternatively use `TRUNCATE TABLE OFFENCE;` rather than using a conditional delete but it does the same job. I would expect that truncate is probably better optimised if dealing with large amounts of data but we've probably only got a few dozen records so thats a non-issue here.

Reply:
That's right . I only use TRUNCATE in exceptional circumstances and sometimes delete doesn't work without a where clause